pages/services/nifi/0.4.0-1.8.0/security/index.md

```diff
@@ -0,0 +1,208 @@
+
+The DC/OS {{model.techName }} service uses a keytab containing the above service and user principals (service keytab). After creating the principals above, generate the service keytab, making sure to include all the node principals. This will be stored as a secret in the DC/OS Secret Store by `name __dcos_base64__secret_name`. The DC/OS security modules will handle decoding the file when it is used by the service. 
+
+<!-- Please make sure this is still current: More details [here.](https://docs.mesosphere.com/services/ops-guide/overview/#binary-secrets) -->
+
+Create secret named "{{ model.serviceName }}admin_kerberos_secret" for password of Kerberos User Principal: `{{ model.serviceName }}admin`
+
+Documentation for adding a file to the secret store can be found [here.](/latest/security/ent/secrets/create-secrets/#creating-secrets-from-a-file-via-the-dcos-enterprise-cli)
+
+<p class="message--note"><strong>NOTE: </strong>Secrets access is controlled by <a href="https://docs.mesosphere.com/latest/security/ent/#spaces-for-secrets">DC/OS Spaces</a>, which function like namespaces. Any secret in the same DC/OS Space as the service will be accessible by the service.</p>
+
+### Install the Service
+Install the DC/OS {{ model.techName }} service with the following options in addition to your own:
+
+   ```json
+   {
+    "service": {
+     "name": "/demo/{{ model.serviceName }}",
+      "security": {
+      "kerberos": {
+           "kdc": {
+          "hostname": "kdc.marathon.autoip.dcos.thisdcos.directory",
+          "port": 2500
+        },
+        "keytab_secret": "__dcos_base64___keytab",
+        "primary": "{{ model.serviceName }}",
+        "realm": "LOCAL",
+        "service_principal": "{{ model.serviceName }}principal@LOCAL",
+        "user_principal": "{{ model.serviceName }}admin@LOCAL",
+        "user_principal_keytab": "{{ model.serviceName }}admin_kerberos_secret"
+      },
+      "kerberos_tls": {
+        "enable": true
+      }
+    },
+    "service_account": "dcos_{{ model.serviceName }}",
+    "service_account_secret": "dcos_{{ model.serviceName }}_secret",
+    "virtual_network_enabled": true,
+    "virtual_network_name": "dcos",
+    }
+  }
+   ```
+
+
+# Configuring DC/OS Access for DC/OS {{model.techName }}
+
+This topic describes how to configure DC/OS access for DC/OS {{model.techName }}. Depending on your security mode, DC/OS {{model.techName }} requires service authentication for access to DC/OS.
+
+
+
+| Security Mode  |   Service Account |
+|----------------|-------------------|
+| Disabled      |    Not available |
+| Permissive    |    Optional |
+| Strict 	     |     Required |
+
+If you install a service in permissive mode and do not specify a service account, Metronome and Marathon will act as if requests made by this service are made by an account with the superuser permission.
+
+### Prerequisites:
+
+- [DC/OS CLI](https://docs.mesosphere.com/1.10/cli/install/) installed and be logged in as a superuser.
```

Comment:
Why are these prerequisites linked to version 1.10 of the Mesosphere docs, rather than later versions?

Reply:
not as such... changed to latest